hr_timesheet_sheet/i18n/fr.po

```diff
@@ -1064,7 +1064,7 @@ msgstr "Semaine"
 #: code:addons/hr_timesheet_sheet/models/hr_timesheet_sheet.py:206
 #, fuzzy, python-format
```

Comment:
remove this line that indicates that was matched through fuzzy

Reply:
Done